src/PointScatter.java

```diff
@@ -9,34 +9,62 @@ public PointScatter() {
     }
 
     public void doRawFitness(Chromo X) {
-        // If we don't code enough locations for two points, the fitness
-        // is not well defined, so reset
         if (Parameters.numGenes < 4) {
             System.out.println("Error: Chromosome must encode at least 2 points");
             System.exit(0);
         }
 
-        // If we don't code an even number of locations, the fitness
-        // is not well defined, so reset
         if (Parameters.numGenes % 2 != 0) {
             System.out.println("Error: Number of genes must be even");
             System.exit(0);
         }
 
         X.rawFitness = Double.MAX_VALUE;
-        for (int i = 0; i < Parameters.numGenes; i += 2) {
-            int x1 = X.getPosIntGeneValue(i);
-            int y1 = X.getPosIntGeneValue(i + 1);
-            for (int j = i + 2; j < Parameters.numGenes; j += 2) {
-                int x2 = X.getPosIntGeneValue(j);
-                int y2 = X.getPosIntGeneValue(j + 1);
-                double d = distanceBetweenPoints(x1, y1, x2, y2);
-                if (d < X.rawFitness) {
-                    X.rawFitness = d;
-                }
-            }
+        
+        
+        if (Parameters.dataRepresentation.equals("xycart")){
```

Comment:
The two options we have, "xycart" and "thetamagpolar" should make their way to our README file. I just want to note this down for future work.